Composer/packages/client/src/components/DropdownWithAllOption/DropdownWithAllOption.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { Dropdown, IDropdownOption, IDropdownStyles } from 'office-ui-fabric-react/lib/Dropdown';
+import React, { useEffect, useState, Fragment } from 'react';
+
+const dropdownStyles: Partial<IDropdownStyles> = {
+  dropdown: { width: 300 },
+};
+
+interface DropdownWithAllOptionProps {
+  optionAllText: string;
+  placeholder: string;
+  selectedKeys: string[];
+  setSelectedKeys: (keys: string[]) => void;
```

Comment:
follow the callback naming pattern instead of imperative function naming:
e.g., onChangeData vs setData